internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1003,6 +1005,32 @@ func urlForRoute(ctx context.Context, asc ASC, routeName string) (string, error)
 	return u.String(), nil
 }
 
+// publicContainerRegistries appends mirror registries and user-specified public registries to the default list
+func publicContainerRegistries(ctx context.Context, asc ASC) string {
+	publicRegistries := []string{"quay.io", "registry.svc.ci.openshift.org"}
+	if asc.spec.MirrorRegistryRef != nil {
+		cm := &corev1.ConfigMap{}
+		if err := asc.rec.Client.Get(ctx, types.NamespacedName{Name: asc.spec.MirrorRegistryRef.Name, Namespace: asc.namespace}, cm); err == nil {
```

Comment:
Do any of the errors here or below get handled? If they're deliberately not being handled, it would help to add a comment about why.

Reply:
Good point, they don't get handled. I'll add a comment, thank you!